### PR TITLE
Merge InterestGroupAction with GroupAction

### PR DESCRIPTION
**Pull request by iselinee**

### CODE REVIEW

File: app/actions/GroupActions.js [hunk 1]
```diff
@@ -205,3 +219,49 @@ export function fetchMembershipsPagination({
     );
   };
 }
+
+export function createInterestGroup(group: Object): Thunk<*> {
```
Comment (LudvigHz) [inline app/actions/GroupActions.js:223]: I think create and remove can also be made generic for groups, and we can just add type as a parameter. (Or you could add a small wrapper with the same args as the old `createInterestGroup` that just sets `type = 'interesse'`.

File: app/actions/GroupActions.js [hunk 2]
```diff
@@ -120,32 +130,36 @@ export function joinGroup(
           role,
         },
         meta: {
-          errorMessage: 'Registrering i gruppe feilet',
-          successMessage: 'Registrering i gruppe fullført',
```
Comment (LudvigHz) [inline app/actions/GroupActions.js:124]: Why remove this?

**Review by LudvigHz — COMMENTED** [review]
Awesome! :rocket: This is definitely a nice improvement
Just some small comments :smile:

File: app/actions/GroupActions.js [hunk 3]
```diff
@@ -94,24 +95,33 @@ export function editGroup(group: Object): Thunk<*> {
...
+            group.type === 'interesse'
+              ? 'Endring av interessegruppe feilet'
+              : 'Oppdatering av gruppe feilet',
+          successMessage:
+            group.type === 'interesse'
+              ? 'Endring av interessegruppe fullført'
+              : 'Oppdatering av gruppe fullført',
         },
       })
+    ).then((_) =>
```
[the remark this answers is not in the document]
Comment (iselinee) [reply app/actions/GroupActions.js:108]: Just copied over from InterestGroupAction, was not sure if it was necessary.

**Review by LudvigHz — APPROVED** [review]
LGTM

**Pull request merged**